### DISCUSSION TITLE
Jump between tunnel ends

### DISCUSSION THREAD

**Discussion by ecraven** [logisim-evolution/logisim-evolution#1757] [Q&A] [answered]
Hello!

Thanks for making an awesome tool available for free ;) I'm trying to capture an old (~80s) design, which uses tunnels all over the place. I'm trying to copy the schematics as-is, so I too have tunnels everywhere. Is there any way to jump between the ends of a tunnel (so between all tunnels with the same name in some sequence)? 

Thanks for any help with this, greetings,
Peter

**Comment by maehne** [comment, marked as answer]
The short answer is no, there currently is no way to jump between the ends of a tunnel using a keyboard/mouse shortcut. We are open to a PR adding this enhancement.

**Reply by ecraven** [reply]
I'll have a look, thank you!

**Reply by ecraven** [reply]
Is there anything like an IRC channel where devs hang out?

**Reply by maehne** [reply]
We primarily communicate through the issue tracker and this forum. Though, there is a Discord channel, which is used by some contributors. @R3dst0ne or @MarcinOrlowski: Could you please provide guidance in this point to @ecraven?

**Reply by R3dst0ne** [reply]
Until now, there haven't been any development related discussions on the Discord. It is mainly used by the community to discuss digital logic design or help with issues how things can be done in LSe.
Here is the [invite link](https://discord.gg/AQFzQJKUcZ) for everyone interested in joining.
